tests/strategies/parse/test_image.py

```diff
@@ -70,37 +74,36 @@ def test_image(
         if failed:
             pytest.skip("Could not find Ghostscript on the system.")
 
-    mime_to_format = {"jpg": "jpeg", "jp2": "jpeg2000"}
-
     sample_file = static_files / filename
     reference_file = (
         static_files / filename_cropped if filename_cropped else static_files / filename
     )
     assert sample_file.exists(), f"Test file not found at {sample_file} !"
     assert reference_file.exists(), f"Test file not found at {reference_file} !"
 
-    for config_parse_test in range(2):
-        print(f"config_parse_test={config_parse_test}")
```

Comment:
This was quite helpful for debugging and understanding when the test failed why it failed. Please leave in.

Reply:
ok